package.json

```diff
@@ -14,7 +14,7 @@
   "homepage": "http://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
-    "test": "grunt test && if [ '$TRAVIS' ]; then grunt coveralls; fi;"
```

Comment:
The fact that using `"` instead of `'` would fix it suggests to me that sticking to JS instead of Bash is the way to go regardless of Windows support. 😄